tests/cpydiff/types_complex_parsing.py

```diff
@@ -0,0 +1,14 @@
+"""
```

Comment:
I suggest calling this file `types_complex_parser.py`, to match `core_fstring_parser.py`.